sentry-sidekiq/lib/sentry/sidekiq/cron/job.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+return unless defined?(::Sidekiq::Cron::Job)
```

Comment:
Since there's no guarantee that the SDK will be loaded AFTER `sidekiq-cron` is required, I think we need to require it here and rescue `LoadError` and return if it fails to load.